src/transmission_manager.rs

```diff
@@ -292,15 +407,15 @@ mod test {
             .clone();
         for seed in 0..n_iter {
             let mut context = setup(1.0);
```

Comment:
Is there a reason for an `r0` of 1.0? Wouldn't that result in only one infection attempt? Am I misunderstanding anything?